jenkins/Jenkinsfile

```
@@ -31,6 +31,30 @@ pipeline {
             '''
           }
         }
+        stage('NDTensorsCUDAExt julia-1.11') {
```

```suggestion
        stage('NDTensorsCUDAExt julia-1') {
```
since if I understand the workflow it will use whatever is the latest release, which at some point will not be 1.11.

---

I updated all of these names. Thanks!